piton/tools/src/sims/sims,2.0

```diff
@@ -2676,6 +2676,7 @@ sub parse_args
             'gui!',
             'log_all!',
             'debug_all!',
+            'debug_pp!',
```

Comment:
What is this one vs debug_all? Can you just use it via `-vcs_build_args=-debug_pp` rather than adding a new flag?